mobile_balance/utils.py

```diff
@@ -10,7 +10,7 @@ def check_status_code(response, expected_code):
         url = response.url
         method = response.request.method
 
-        raise BadResponse(('{method} to {url} resulted '
+        raise BadResponse((f'{method} to {url} resulted '
```

Comment:
Please, remove `.format(**locals())` as well.

Reply:
Also, f' prefix should be added to each string on each line. Otherwise `{result_code}` and `{expected_code}` will not replaced:

```
In [1]: foo = 1

In [2]: bar = 2

In [3]: print(f'Foo is {foo}. '
   ...:        'Bar is {bar}')
Foo is 1. Bar is {bar}
```

Reply:
Сейчас провайдер Мегафон внес изменения и модуль на данного провайдера  вцелом не рабочий.

Reply:
> Also, f' prefix should be added to each string on each line. Otherwise `{result_code}` and `{expected_code}` will not replaced:
> 
> ```
> In [1]: foo = 1
> 
> In [2]: bar = 2
> 
> In [3]: print(f'Foo is {foo}. '
>    ...:        'Bar is {bar}')
> Foo is 1. Bar is {bar}
> ```

Yes! 

The megaphone module is working in this repository. Provider made changes